contracts/margin/external/exchangewrappers/ZeroExV2ExchangeWrapper.sol

```diff
@@ -0,0 +1,295 @@
+/*
+
+    Copyright 2018 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.4.24;
+pragma experimental "v0.5.0";
+pragma experimental ABIEncoderV2;
```

Comment:
Do we need to use this to interact with 0x's contracts? We just don't use it anywhere else

Reply:
I see that `FillResults memory fill = v2Exchange.fillOrKillOrder` returns a struct so I'm guessing we do

Reply:
Yeah otherwise I get the error:
`InternalCompilerError: Static memory load of more than 32 bytes requested.`